content/docs/2.13/troubleshooting.md

```diff
@@ -3,4 +3,38 @@ title = "Troubleshooting"
 description = "How to address commonly encountered KEDA issues"
 +++
 
+## Troubleshoot keda errors using profiling
+
+in golang we have the possibility to profile specific actions in order to determine what causes an issue.
+For example, if our keda-operator pod is keeps getting OOM after a specific time, using profilig we can profile the heap and see what operatios taking all of this space.
+
+Golang support many profiling options like heap, cpu, goroutines and more... (for more info check this site https://pkg.go.dev/net/http/pprof).
+
+In keda we provide the option to enable profiling on each component separately by enabling it using
+the helm chart and providing a port (if not enabled then it won't work)
+
+```yaml
+profiling:
+  keda:
+    enabled: false
+    port: 8082
+  metricsApiServer:
```

Comment:
please also incorporate changes from the charts here

Reply:
Can I just create it and bring reference to the keda docs?

Reply:
Fixed :)